vue_frontend/src/router/index.js

```diff
@@ -0,0 +1,25 @@
+import { createRouter, createWebHistory } from 'vue-router'
+import Home from '../views/Home.vue'
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: Home
+  },
+  {
+    path: '/about',
+    name: 'About',
+    // route level code-splitting
+    // this generates a separate chunk (about.[hash].js) for this route
+    // which is lazy-loaded when the route is visited.
+    component: () => import(/* webpackChunkName: "about" */ '../views/About.vue')
```

Comment:
I think it would be a good idea to try to be consistent with your code. By that I mean: you're using two different approaches to import Components to your `routes`. Try sticking with one of them.

Reply:
This About component will probably be removed on the next PR, but I will also keep that in mind when creating new components.